### PR TITLE
DustID

### PR DESCRIPTION
**Pull request by SmugBlanco**
Current state:
DustID is barebones and barely utilized due to that fact. You would often have to refer to the popular forum page 'Dust and Sound Catalogue 2' just to get a semi accurate image of what the dust may be used for.

Proposed changes:
This, a more flushed out DustID class who's variables are named after my best guess of it's original implementation purpose or most popular usage of it.

### CODE REVIEW

**Review by Mirsario — CHANGES_REQUESTED** [review]
I should've told you the following in discord right away:
- Additions should be in DustID.TML.cs partial file.
- Don't remove or rename vanilla entries.
- There's no need for line breaks, but you can have them in the partial file, outside patches.

**Comment by SmugBlanco** [comment]
My bad, undid and the changes to vanilla as you said and made the changes into a partial class.

File: patches/tModLoader/Terraria.ID/DustID.TML.cs [hunk 1]
```diff
@@ -0,0 +1,322 @@
+﻿namespace Terraria.ID
+{
+	public static partial class DustID
+	{
+		//Naming is based on the best approximation of earliest implimentation/most popular
+		public const short Grass_Small = 3;
```
Comment (direwolf420) [inline patches/tModLoader/Terraria.ID/DustID.TML.cs:6]: > implimentation

Comment (Mirsario) [reply patches/tModLoader/Terraria.ID/DustID.TML.cs:6]:
```suggestion
	{
		//Naming is based on the best approximation of the earliest or most popular implementation.
		public const short Grass_Small = 3;
```

**Review by Mirsario — APPROVED** [review]
I had a complaint about underscores, but democracy happened:

**Pull request merged**